README.md

```diff
@@ -0,0 +1,67 @@
+# BackTREK
+
+## Introduction
+
+For this project we'll be returning to the TREK API, using Backbone to build an application that can handle data in complex ways.
+
+## Learning Goals
+
+- Use Backbone and jQuery to work with a complex API
+- Manage application data using Backbone models and collections
+- Build an attractive, robust, and feature-rich user interface
+
+## Wave 1
+
+In wave 1, you'll use our new functionality (Underscore templates, Backbone models and collections) to achieve feature parity with the original TREK project, including some of the optionals:
+
+- A user can click a link to see a list of trips
+- A user can click on a trip to see details of that trip
+- A user can create a new trip
```

Comment:
I guess I haven't seen in the notes yet where we'll be showing them how to do this. Will this be part of the events on collections material?

Reply:
By "this", do you mean creating a new trip? That should be covered as the first part of the events lecture.

https://github.com/Ada-Developers-Academy/textbook-curriculum/pull/339/files#diff-5966e4abc47bfbcdc66fe5c172075363

Reply:
Ok - thanks